frontend/src/openapi/models/personalDashboardSchemaProjectsItem.ts

```diff
@@ -4,7 +4,32 @@
  * See `gen:api` script in package.json
  */
 
+export type SystemOwner = { ownerType: 'system' };
```

Comment:
is everything here auto generated?

Reply:
No. it's hand-crafted and copied. I didn't wanna generate orval just yet, but the schema is in, so it should be possible to do. lemme have a look.